server/api.js

```diff
@@ -84,4 +85,88 @@ router.get("/quizes",(req, res)=>{
 	});
 });
 
+router.get("/quizes/search",(req, res)=>{
+	const client = getClient();
+	client.connect(()=>{
+		const db = client.db("quiz");
+		const collection = db.collection("quizes");
+		const quiz = { quiz_name:req.query.quiz_name };
+		collection.findOne(quiz,(err, questions)=>{
+			if(err){
+				res.status(500).send(err);
+			}else{
+				res.status(200).send(questions);
+			}
+		});
+	});
+});
+
+router.get("/results/search",(req, res)=>{
```

Comment:
This is finding all results for a given quiz?

Reply:
yes, search for all results by specific quiz name.